protobuf.bzl

```diff
@@ -655,6 +660,7 @@ def _source_proto_library(
         protoc = Label("//:protoc"),
         testonly = None,
         visibility = ["//visibility:public"],
+        enable_editions = False,
```

Comment:
did you mean to add this to the php macro?

Reply:
Without this, I get `name 'enable_editions' is not defined`. Is there a better way?

Reply:
whoops nvm